dependencies.yaml

```
@@ -249,3 +255,19 @@ dependencies:
           - matrix:
             packages:
               - *librmm_unsuffixed
+
+  depends_on_nccl:
+    common:
+      - output_types: conda
+        packages:
+          - &nccl_unsuffixed nccl>=2.19
```

```suggestion
          - nccl>=2.19
```

This `nccl_unsuffixed` alias isn't getting used anywhere... it should be removed.